ui/webui/resources/polymer_overriding.html

```diff
@@ -18,53 +18,47 @@
   <template>
     <style>
       /* Default/light theme button colors */
-      :host {
+      :host(:not([style])) {
         --brave-text-color: #3b3e4f; /* grey800 */
         --brave-brand-color: #ff7654; /* orange400 */
         --brave-primary-hover: #ff977d; /* orange300 */
         --brave-default-hover: rgba(255, 118, 84, 0.1); /* orange400 */
         --brave-focus-outline: rgba(255, 118, 84, 0.4); /* orange400 */
-        --brave-default-border: #c2c4cf; /* grey 400 */
         --ink-color: var(--brave-brand-color) !important;
+        --text-color: var(--brave-text-color) !important;
+        --text-color-action: white !important;
+        --border-color: #c2c4cf !important; /* grey 400 */
+        --active-shadow-rgb: 255,118,84 !important;
+        --active-shadow-action-rgb: 255,118,84 !important;
+        --bg-action: var(--brave-brand-color) !important;
+        /* TODO: Disabled colors:
```

Comment:
should this comment be removed?

Reply:
If you think so. My intention was to keep it so that it was clear that we have not overriden any colors for the disabled state.

Reply:
ah ok - that seems fine 😄